app/static/common/markdown.css

```
@@ -0,0 +1,60 @@
+/* Following copied from inspector on github.com */
+body {
+  background-color: var(--background-color);
+  color: var(--text-color);
+
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji';
+  font-size: 14px !important;
+  line-height: 1.5;
+  word-wrap: break-word;
+}
+
+code, tt {
+  padding: .2em .4em;
+  margin: 0;
+  font-size: 85%;
+  background-color: var(--code-background-color);
+  border-radius: 6px;
+}
+
+table {
+  display: block;
+  width: 100%;
+  width: -webkit-max-content;
+  width: -moz-max-content;
+  width: max-content;
```

We don't need vendor-specific prefixes, we know which browser we're on

```suggestion
  width: max-content;
```

---

:D copy/pasta -  Good point. Like to also point out that this css was all just a quick copy/pasts from dotcom and likely in use we will add/modify it. For example, when working my hack day project, I found it was useful for image widths to be set to 100% so they would respect an outer width set to the iframe.

---

Seems like this was marked as resolved without the change being made?